composer.json

```diff
@@ -44,6 +44,7 @@
         "symfony/yaml": "~2.3",
         "symfony/security-acl": "~2.4",
         "sonata-project/intl-bundle": "~2.1",
+        "sllh/php-cs-fixer-styleci-bridge": "~1.0",
```

Comment:
>  styleci's symfony preset is not the same as php-cs-fixer's symfony level

Why having a different preset? This is confusing. Maybe this should be on your `recommended` preset.

Could you please elaborate the diff?